doc/changes.md

```diff
@@ -1,5 +1,75 @@
 # Changelog
 
+## Release v0.3.1 (2021/03/14)
+
+[![Digital Object Identifier for PyGMT v0.3.1](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.3781524)
+
+### Highlights
+
+* 🎉 **Multiple bug fixes and a reorganized gallery** 🎉
+* New gallery example for Cartesian, circular, and geographic vectors ([#950](https://github.com/GenericMappingTools/pygmt/pull/950))
```

Comment:
> Then I'd like to recommend another gallery example: [Vector heads and tails](https://www.pygmt.org/dev/gallery/lines/vector_heads_tails.html#sphx-glr-gallery-lines-vector-heads-tails-py). It is cool and useful. How do you think ?@weiji14 @willschlitzer

Yep, actually #950 and #890 are part of the vectors issue at #892 so maybe we could merge it into one bullet point?

For another point, how about mentioning the gallery reorganization #995? There was quite a bit of work (and debate) that went into that!

Reply:
> Yep, actually #950 and #890 are part of the vectors issue at #892 so maybe we could merge it into one bullet point?

Sounds good.

> For another point, how about mentioning the gallery reorganization #995? There was quite a bit of work (and debate) that went into that!

I agree, but the first line says:

> 🎉 **Multiple bug fixes and a reorganized gallery** 🎉

So can we still highlight it again?

Reply:
Added in 444ef30

Reply:
> I agree, but the first line says:
> 
> > tada **Multiple bug fixes and a reorganized gallery** tada
> 
> So can we still highlight it again?

Or we can change that first line to say ":tada: Multiple bug fixes and an improved gallery :tada:" :shrug:

Reply:
Alternatively, we can mention the Twitter account :bird:https://twitter.com/gmt_software

Reply:
> Or we can change that first line to say " Multiple bug fixes and an improved gallery "

That's great. Will update https://hackmd.io/@pygmt/ryt7DI5md when other maintainers approve this PR.

Reply:
Great, thanks for starting that changelog @core-man! I've added a nice link so using https://hackmd.io/@pygmt/v0-3-1_announcement now:

![image](https://user-images.githubusercontent.com/23487320/111063838-b3116080-8515-11eb-8811-218f1495efc9.png)

Reply:
From my point of view, the most important changes are covered 😉.